src/drivers/sf1xx/sf1xx.cpp

```diff
@@ -594,24 +593,17 @@ SF10A::cycle()
 		return;
 	}
 
-
-	/* Trigger measurement */
-	if (OK != measure()) {
-		DEVICE_DEBUG("measure error lidar");
-	}
-
```

Comment:
Yes, i've tested it by hot-remove and hot-plug sensor and ensure that uORB announcements restarts back.